ironfish-cli/src/utils/language.ts

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+import { LanguageCode } from '@ironfish/rust-nodejs'
+import inquirer from 'inquirer'
+
```

Comment:
Unfortunately we need to re-export the constant here because the export from napi is `const enum` which can only be referenced as values